bool print_map
---
full_coverage/CellGrid cell_grid
